### PR TITLE
feature(footer)

### PR DESCRIPTION
**Pull request by Hanouj**
# Description

Please provide a short summary explaining what this PR is about.

Fixes # (issue)

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: packages/client/src/components/Footer/FooterStyle.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  font-size: large;
+}
+.icon {
+  height: 23px;
+  width: 33px;
+  border: 4px solid rgb(9, 9, 9);
+  border-radius: 50%;
+  background-color: rgb(252, 219, 219);
+  box-shadow: 0 10px 10px rgba(0, 0, 0, 0.1);
+  cursor: pointer;
```
Comment (shpomp) [inline packages/client/src/components/Footer/FooterStyle.css:68]: The border should not be there. The icons go in as they are. 
Pleas remove for the next iteration. 🙏🏽

**Review by shpomp — APPROVED** [review]
👍🏽 ⭐

File: packages/client/src/components/Footer/FooterStyle.css (hunk 1 above)
Comment (Hanouj) [reply packages/client/src/components/Footer/FooterStyle.css:68]: Okay 👍🏼

**Review by Hailegebriel22 — COMMENTED** [review]
@Hanouj it looks fine.

**Review by shpomp — DISMISSED** [review]

**Review by shpomp — APPROVED** [review]

**Review by Hailegebriel22 — APPROVED** [review]

**Pull request merged**